packages/billing/src/utils/paymentSplitter.ts

```diff
@@ -1,50 +1,52 @@
-const { Big } = require('big.js')
-const { get, set, omit } = require('lodash')
-
-/**
- * @typedef {Object} TRecipient
- * @property {string} bankAccount
- * @property {string} [bankName]
- * @property {string} bic
- * @property {string} [classificationCode]
- * @property {string} [iec]
- * @property {string} [name]
- * @property {string} [offsettingAccount]
- * @property {string} [territoryCode]
- * @property {string} tin
- */
+import { Big } from 'big.js'
+import { get, set, omit } from 'lodash'
```

Comment:
- This import is banned. Consider using individual imports instead
- `get` in typescript is also not welcomed, since you've got `?.` for a long time now.
- `set` / `omit` is fine I guess, but if we can remove it - I'll be happy (optional)

Reply:
Yes, fixed